i18n/po/template.pot

```diff
@@ -3104,6 +3221,14 @@ msgstr ""
 msgid "Your bitcoin wallet is ready!"
 msgstr ""
 
+#: www/views/modals/chooseFeeLevel.html:38
+msgid "Your could not set a fee higher than 1000 satoshis/byte."
+msgstr ""
+
+#: www/views/modals/chooseFeeLevel.html:34
+msgid "Your fee is lower than the recommended super economy fee ({{getMinimumRecommeded()}} satoshis/byte). Your transaction may never get confirmed."
```

Comment:
Should be: "lower than recommended..." and "The transaction may never get confirmed"